m3u8/version_matching_rules.py

```diff
@@ -0,0 +1,44 @@
+from m3u8 import protocol
+
+
+# You must use at least protocol version 2 if you have IV in EXT-X-KEY.
+def valid_iv_in_EXT_X_KEY(line: str, version: float):
+    if not protocol.ext_x_key in line:
+        return True
+
+    if "IV" in line:
+        return version >= 2
+
+    return True
+
+
+# You must use at least protocol version 3 if you have floating point EXTINF duration values.
```

Comment:
maybe use this tip/description attached to the error helping users to know how to fix their problems

Reply:
Got it!